server/lib/chat/chat-service.test.ts

```diff
@@ -87,6 +89,37 @@ jest.mock('./chat-models', () => ({
   searchChannelsAsAdmin: jest.fn().mockResolvedValue([]),
 }))
 
+// TODO(2Pac): Move these elsewhere?
+
+type DbJoinChannelMessage = ChatMessage & { data: JoinChannelData }
```

Comment:
Call these like `FakeDbFoo`, calling them `DbJoinChannelMessage` is confusing since that's how we name stuff in our model files and these are not what those are